scripts/simulation/experience.ts

```diff
@@ -0,0 +1,70 @@
+import { strict as assert } from "assert";
+import chalk from "chalk";
+
+import { SimulatorContext } from "../simulator";
+
+import { withErrorReporter } from "../lib/log";
+import { addBotReaction as actualReactToExperience } from "../lib/bot";
+import { sleep } from "../lib/utils";
+
+export const DEFAULT_EXPERIENCE_CHUNK_SIZE = 100;
+export const DEFAULT_EXPERIENCE_TICK_MS = 1000;
+export const DEFAULT_EXPERIENCE_AFFINITY = 0.005;
```

Comment:
I have taken care to compartmentalize simulation specific constants into their own files as to not create unnecessary references.